packages/editor/src/components/blocks/input/Input.tsx

```diff
@@ -36,15 +50,28 @@ export const InputComponent: ComponentConfig<InputProps> = {
     label: { subsection: 'General', label: 'Label', type: 'textBrowser', browsers: ['CMS'] },
     value: { subsection: 'General', label: 'Value', type: 'textBrowser', browsers: ['ATTRIBUTE'] },
     type: { subsection: 'General', label: 'Type', type: 'select', options: typeOptions },
+    decimalPlaces: { subsection: 'Formatting', label: 'Decimal Places', type: 'number', hide: data => data.type !== 'NUMBER' },
+    symbol: { subsection: 'Formatting', label: 'Symbol', type: 'text', hide: data => data.type !== 'NUMBER' },
+    symbolPosition: {
```

Comment:
maybe this stuff should live in it's own accordion... only number specific configs

Reply:
Yes, maybe. How would you name the accordion 'Number'? However, when I think about these settings, they are still just properties that are being set, but I can move it into its own accordion for now.

Reply:
I think I merge everything as is (without a separate accordion) and quickly review it with Bruno tomorrow. I think it is also fine to just add the collapsible, as long as the other input types don’t have specific properties. If this input grows more complex, we can always expand it later ✌️